### DISCUSSION TITLE
https://sredevops.org/en/lima-the-easiest-way-to-run-any-linux-distro-kubernetes-k3s-and-even-docker-on-macos-and-linux-apple-silicon-m1-arm64-compatible/

### DISCUSSION THREAD

**Discussion by ghost** [sredevopsorg/.github#54] [Comentarios en SREDevOps.org]
# https://sredevops.org/en/lima-the-easiest-way-to-run-any-linux-distro-kubernetes-k3s-and-even-docker-on-macos-and-linux-apple-silicon-m1-arm64-compatible/

Lima is a versatile and user-friendly command line tool (CLI) that empowers you to seamlessly run Linux virtual machines (VMs) on your macOS or Linux system. It's compatible with any Apple Silicon Mac (M1, M2, etc) ARM64 and Intel x86_64 processors, vice versa, without anything else than a single

https://sredevops.org/en/lima-the-easiest-way-to-run-any-linux-distro-kubernetes-k3s-and-even-docker-on-macos-and-linux-apple-silicon-m1-arm64-compatible/

**Comment by ngeorger** [comment]
<3
